sys/linux/auto.txt

```diff
@@ -277,7 +277,7 @@ futex_time64$auto(uaddr intptr, op intptr, val intptr, utime intptr, uaddr2 intp
 futex_wait$auto(uaddr intptr, val intptr, mask intptr, flags intptr, timeout intptr, clockid intptr) (automatic)
 futex_waitv$auto(waiters intptr, nr_futexes intptr, flags intptr, timeout intptr, clockid intptr) (automatic)
 futex_wake$auto(uaddr intptr, mask intptr, nr intptr, flags intptr) (automatic)
-futimesat$auto(dfd intptr, filename intptr, utimes intptr) (automatic)
+futimesat$auto(dfd intptr, filename intptr, t intptr) (automatic)
```

Comment:
What has caused this change? The PR has only netlink-related changes, right?
Is the tool output non-deterministic (changes on every re-run)?

Reply:
Hmmm. I'll look more into it. Maybe it's related to one of the Go code changes in the last PR, but went unnoticed.

Reply:
Fixed! Now parameter names are also taken into consideration while sorting, ensuring that we keep the same system call copy every time.